[core] Batch small changes (#17807)
mui-org/material-ui: docs/src/pages/components/material-icons/material-icons.md

includes the 1,000+ official [Material icons](https://material.io/tools/icons/?style=baseline) converted to [`SvgIcon`](/api/svg-icon/) components.

{{"demo": "pages/components/material-icons/SearchIcons.js", "hideHeader": true}}

> ℹ️ The search supports synonyms. Try a search with "hamburger", or "logout".

Reviewer: ```suggestion
ℹ️ The search supports synonyms. Try searching for "hamburger", or "logout".
```
?

> Tip: the search supports synonymes, try a search with "hamburger" or "logout".

Reviewer 2: ```suggestion
Tip: the search supports synonyms. Try a search with "hamburger", or "logout".
```
